src/packages/DAO/elections.dao.ts

```diff
@@ -1,12 +1,19 @@
 "use server";
 
 import prisma from "@lib/prisma";
-import { createElectionValidator } from "@packages/dto/elections.dto";
+import {
+  createElectionValidator,
+  electionSchema,
+  updateElectionSchema,
+} from "@packages/dto/elections.dto";
 import type { CreateElection } from "@packages/entities/elections";
+import { BadRequestError, NotFound } from "@packages/utils/error";
 import { ElectionStatus } from "@prisma/client";
 import { revalidatePath } from "next/cache";
 import { redirect } from "next/navigation";
-import { z } from "zod";
+import type { z } from "zod";
+
+type FindElection = z.infer<typeof createElectionValidator>;
```

Comment:
I think we can move these lines to the `@packages/dto/elections.dto`

Reply:
change here [83cb29b](https://github.com/podcodar/voting-system/pull/67/commits/83cb29b9cca4cd1a378324ef34db53c1c98d8d79)